src/V8JsiRuntime.cpp

```diff
@@ -1574,7 +1667,69 @@ v8::Local<v8::Value> V8Runtime::valueRef(const jsi::Value &value) {
   }
 }
 
+void applyOverrides(V8RuntimeArgs &args) {
+  constexpr const wchar_t *regSubKey =
+      L"SOFTWARE\\Microsoft\\V8JSI\\";
+  DWORD value, bufferSize = sizeof(DWORD);
+
+  if (ERROR_SUCCESS ==
+      RegGetValueW(
+          HKEY_CURRENT_USER,
+          regSubKey,
+          L"enableInspectorOverride",
+          RRF_RT_REG_DWORD,
+          nullptr,
+          static_cast<void *>(&value),
+          &bufferSize)) {
+    args.enableInspector = value > 0;
+  }
+
+  if (args.enableInspector) {
+    if (ERROR_SUCCESS ==
+        RegGetValueW(
+            HKEY_CURRENT_USER,
+            regSubKey,
+            L"inspectorPortOverride",
+            RRF_RT_REG_DWORD,
+            nullptr,
+            static_cast<void *>(&value),
+            &bufferSize)) {
+      args.inspectorPort = value;
+    }
+
+    if (ERROR_SUCCESS ==
+        RegGetValueW(
+            HKEY_CURRENT_USER,
+            regSubKey,
+            L"waitForDebuggerOverride",
+            RRF_RT_REG_DWORD,
+            nullptr,
+            static_cast<void *>(&value),
+            &bufferSize)) {
+      args.waitForDebugger = value > 0;
+    }
+  }
+
+  if (ERROR_SUCCESS ==
+      RegGetValueW(
+          HKEY_CURRENT_USER,
+          regSubKey,
+          L"enableDetailedTracing",
+          RRF_RT_REG_DWORD,
+          nullptr,
+          static_cast<void *>(&value),
+          &bufferSize)) {
+    const bool enable = value > 0;
+    args.trackGCObjectStats = enable;
+    args.enableTracing = enable;
+    args.enableJitTracing = enable;
+    args.enableMessageTracing = enable;
+    args.enableGCTracing = enable;
+  }
+}
+
 std::unique_ptr<jsi::Runtime> makeV8Runtime(V8RuntimeArgs &&args) {
+  applyOverrides(args);

```

Comment:
>applyOverrides(args); [](http://example.com/codeflow?start=2&length=21)

Can you submit the rest of the changes besides this registry functionality (I think this is the only sticking point of the PR and we can revisit it later). There's a formatting PR in the pipeline and merging that would be painful after the fact.